lib/crypto/crypto_keys_subr.c

```diff
@@ -180,23 +183,23 @@ crypto_keys_subr_import_RSA_priv(RSA ** key, const uint8_t * buf, size_t buflen)
 	if (import_BN(&n, &buf, &buflen))
 		goto err1;
```

Comment:
It's the first one, so if it failed, no memory was allocated?  I'm happier to make it `err2` for consistency, but I figured you'd object to that.